server/server.go

```diff
@@ -137,6 +137,8 @@ type Query struct {
 	OrTags []QueryTag
 	// How many traces to return. Zero means no limit.
 	Num int
+	// ServiceName to filter by.
```

Comment:
Nitpick, but did you mean to write "// ServiceName**s** to filter by." here? Since the variable is called `ServiceNames`. Or was the removal of "s" intentional?